### ISSUE TITLE
Equipping AdvancedAE Quantum Armor with Wireless Charging upgrades generates 2.1 GFE/s for the AE network with AppliedFlux cells while holding Digital Miner

### ISSUE THREAD

**Issue by Kribylet** [mekanism/Mekanism#8273]
### Issue description

This issue is a cross-post from the AdvancedAE github after the author investigated the issue:
https://github.com/pedroksl/AdvancedAE/issues/66

This occurs in the DW20 1.4.1 (and 1.8.0) pack. My energy source is an Extreme Reactor generating 18~ KFE/t (5x5x5 core, Redfrigium, passive) set up to feed an energy acceptor and flux acceptor immediately at the charge port through 1 Ultimate Universal Cable (Mekanism).

When Wireless mode is enabled on the quantum armor, my energy generation spikes by 2.1 trillion FE/s while holding a Digital Miner.

Holding a Digital Miner in isolation while wearing the Quantum Armor caused the energy generation spike to occur. Holding an Ultimate Energy Cube, 1 fully upgraded Mining Gadget, 1 atomic disassembler and 1 Armored Free Runners but no Digital Miner the energy spike did not occur. I had previously had it deployed with a Quantum Entangloporter (Mekanism) linking it for power in the mining dimension, not sure if there is some related state there or not.

Tangentially, we've found another issue related to the digital miner on our MP server where it is valid as a recharging object in the ChargingGadgets mod charging station. Attempting to recharge it in the charging station of that mod causes a server-wide freeze that requires a manual restart.

### Steps to reproduce

![energy_ae_setup](https://github.com/user-attachments/assets/574936c7-e9c0-4186-9064-6ef04c55dcac)
![digital_miner_setup_before_teardown](https://github.com/user-attachments/assets/cccb70fd-32e3-48ba-bc89-b457f533f868)
![energy_ae_setup_holding_digital_miner](https://github.com/user-attachments/assets/03789b41-1bd5-4673-8742-79e42482008b)
![energy_cells_filling](https://github.com/user-attachments/assets/415422ae-e054-4b59-ac2a-eaa98c8f2f3d)

Here is the setup I created in singleplayer to repro this issue

1. Set up reactor in overworld
2. AE network: ME Controller, Energy Cell, ME Drive + 9 256M FE Storage Cells
3. AE-Reactor link: Ultimate Universal Energy Cable, Flux Accessor, Energy Acceptor to AE net
4. Quantum Entangloporter linked with Flux Accessor, set to private net
5. Go to mining dimension
6. Set up digital miner with 32 energy upgrades, 16 speed upgrades
7. Set up entangloporter and link power by ultimate universal cable
8. Wait for digital miner to fully charge
9. Take down digital miner using Configurator in wrench mode, then tear down ultimate cable and entangloporter
10. Wear full unlinked Quantum Armor set
11. Equip wireless upgrade on each part
12. Set up wireless access point on AE network, link each armor piece
13. Wait for armor to fully charge (I sped this up with a SolarGeneration Ultimate Solar Panel in this case)
14. As armor fully charges, the energy spike occurs.

I also tried tearing down the energy link cable and entangloporter before tearing down the digital miner, with the same results.

### Minecraft version

1.21.1 (Latest)

### NeoForge version

21.1.74

### Mekanism version

10.7.7 (Latest)

### Other relevant versions

AdvancedAE-1.1.1-1.21.1

Also repro in AdvancedAE-1.1.12-1.21.1 (manually applied to DW20 1.8.0)

**Comment by thiakil** [comment]
The digital miner is a valid target for being charged when in item form.

It's unlikely that it's at fault tho. Please try reproduce with just Mekanism and Advanced AE

**Comment by pedroksl** [comment]
To add to this, I tried reproducing this issue with a minimal setup and couldn't do it. I wonder if more items in inventory are needed to enable this weird interation. Something might be looping energy and creating more in the process.

**Comment by thiakil** [comment]
Try adding the mod that gives extra energy upgrades. It messes with internals.

**Comment by pedroksl** [comment]
Adding that mod made me able to reproduce.... but only once. I can't anymore, even when doing the exact same steps. It also only happened with 16 energy upgrades, but not with 32. I think it's safe to say this issue is not related to base Mekanism, but rather the addon.

**Comment by pedroksl** [comment]
Hi. As discussed here:
https://github.com/WhitePhant0m/Mekanism-Unleashed/issues/2

The dev for Mekanism-Unleashed was able to reproduce the infinite power generation with just mekanism config changes. It seems that is actually related to Mekanism, while the item max power issues are related to his mod.

**Comment by thiakil** [comment]
Please create a separate issue with reproduction steps for that
